docs/data/material/customization/css-theme-variables/default-tokens-viewer.md

```diff
@@ -0,0 +1,63 @@
+# Default tokens viewer
+
+<p class="description">A preview of the default CSS theme variables for built-in light and dark color schemes</p>
+
+The following tokens are generated when using `CssVarsProvider` in your application.
+
+## Colors
+
+The color tokens are generated from `colorSchemes`:
+
+{{"component": "modules/material-ui/PaletteTokensViewer.js"}}
+
+## Font
+
+The font tokens are generated from `typography`:
+
+{{"component": "modules/material-ui/FontTokensViewer.js"}}
+
+:::success
+Using font variable is an alternative way to `theme.typography` when you cannot access the theme in Javascript.
+The font variable should be used with the CSS `font` property like this:
+
+```css
+.text {
+  font: var(--mui-font-h1);
+}
+```
+
+:::
+
+## Spacing
+
+The variable `--mui-spacing` is generated from the `spacing` field with a default of `8px`.
+
+Below are the usage examples:
+
+<codeblock>
+
+```js JS
+theme.spacing(1, 2); // var(--mui-spacing) calc(2 * var(--mui-spacing))
+```
+
+```css CSS
+.some-class {
+  margin: calc(var(--mui-spacing) * 2);
+}
+```
+
+</codeblock>
+
+## Shadows and overlays
```

Comment:
Q: are me missing a section in the docs for this in the Customization > Tokens section?

I've been having some trouble customizing the shadows and couldn't find much about it in the docs. The only page I found was under [MUI System](https://mui.com/system/shadows/), which isn't ideal. I agree that we should have a page for it—especially since we have 25 shadow tokens, the least we could do is show them off 😅

cc @samuelsycamore

Reply:
Just to be clear, it's out-of-scope of this PR right?